### PR TITLE
[Meetingsapps][Responsebody][1609300]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Incorporated changes for issues 2823, 2873 and 2856. Please review.

### CODE REVIEW

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -322,3 +323,9 @@ The post-meeting and pre-meeting configurations are equivalent.
 
  > [!div class="nextstepaction"]
 > [Meeting token generator app](https://github.com/OfficeDev/microsoft-teams-sample-meetings-token)
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:325]: Could you please remove "Meeting app sample" section and add link to code in Code Samples section? Code samples section would look like this:
image
Source: https://github.com/OfficeDev/microsoft-teams-samples#scenario-specific-samples
Link to use: https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-token-app/csharp

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Teams’ meeting app extensibility centers on three concepts:
 
 ![pre-meeting tab view](../assets/images/apps-in-meetings/PreMeetingTab.png)
 
-✔ Permissioned users can add apps to a meeting via the tab gallery in two ways:
+✔ Permissioned users are users who can add tabs to a meeting during different stages of the meeting lifecycle. These users can add apps to a meeting via the tab gallery in two ways:
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:41]:
```suggestion
✔ Permissioned users are users who can add apps to a meeting during different stages of the meeting lifecycle. These users can add apps to a meeting through the tab gallery in two ways:
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please do a thorough self-review.

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 3]
```diff
@@ -103,35 +111,51 @@ For messaging extension implementation, start with [build a messaging extension]
...
-1. **Federated/External**. A federated user is an external Teams user in another organization who has been invited to join a meeting. Since these users have valid credentials with federated partners, they are treated as authenticated by Teams but do not have access to your teams or other shared resources from your organization. If you want external users to have access to teams and channels, guest access might be a better option. _See_ [Manage external access in Microsoft Teams](/microsoftteams/manage-external-access)
-1. **Anonymous**. Anonymous users do not have an Active Directory identity and are not federated with a tenant. The anonymous participant is like an external user, but their identity is not projected into the meeting. Anonymous users will not be able to access apps in a meeting window.
+The following user types identify what each user can do and what they can access:
 
-## Next steps
+* In-tenant: In-tenant users belong to the organization and have credentials in Azure Active Directory (AAD) for the tenant. They are usually full-time, onsite, or remote employees. An in-tenant user can be an organizer, presenter, or attendee.
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:139]: Can we make these bold so that it stands out?

**Review by Wajeed-msft — APPROVED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md (hunk 3 above)
Comment (v-rodriguezd) [reply msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:139]: As per MSTP ideally we only bold User interface elements

**Review by KirtiPereira — DISMISSED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved.

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**